test/generator.test.js

```diff
@@ -383,4 +383,59 @@ describe('Generator', () => {
       expect(content).toBe(utils.__files[filePath]);
     });
   });
+
+  describe('#loadDefaultValues', () => {
+    it('default value of parameter is set', async () => {
+      const gen = new Generator('testTemplate', __dirname, {
+        templateParams: {
+          test: true
+        }
+      });
+      const json = '{"parameters": {' +
```

Comment:
JS supports template literals and I recommend it here instead of concatenating strings like this

Reply:
anyway why string?

Reply:
done